RemoveLinksToAttachments.py

```diff
@@ -2,11 +2,14 @@
 Redmine Support #4173
 Removes all attachments and references/links to attachments from the schema by selecting ID's from collection objects
```

Comment:
description should make it clear that the script removes attachments where the MIME type is `image/jpeg`, not _all_ attachments.

Reply:
Banner has been rewritten to make it clear that the script removes the attachments with MimeType as image/jpegs
https://github.com/AAFC-BICoE/specify-scripts/blob/f1c1ce7bae43fde6c4cef3543d95ead5bdbd412d/RemoveLinksToAttachments.py#L2